source/draw/Image.ooc

```diff
@@ -69,7 +69,7 @@ Image: abstract class {
 		super()
 	}
 	resizeWithin: func (restriction: IntVector2D) -> This {
-		this resizeTo(((this size toFloatVector2D()) * Float minimum(restriction x as Float / this size x as Float, restriction y as Float / this size y as Float)) toIntVector2D())
+		this resizeTo(((this size toFloatVector2D()) * (restriction x as Float / this size x as Float) minimum(restriction y as Float / this size y as Float)) toIntVector2D())
```

Comment:
To be honest one can argue that previously this line was slightly more readable. What do you think ?